### DISCUSSION TITLE
Please add the OpenGL max zoom level to Software renderer

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#20888] [Feature requests] [answered]
I don't know how this could be implemented, but I find it jarring using Software because it lacks the closest zoom level. My work with tile inspector is drastically effected. Please add the OpenGL highest zoom level to Software. Thank you!!

**Comment by duncanspumpkin** [comment]
It was purposely removed as we couldn't work out how to get it to work. It would cause sprites to not have the same location every render pass causing lots of glitchy images.

**Reply by 733737** [reply]
does this mean it will not be revisited at this time?

**Reply by duncanspumpkin** [reply, marked as answer]
It's an open source project people work on whatever they feel like. I don't know of anyone interested in this particular part at this time but who knows maybe someone is.

**Comment by 733737** [comment]
It was implemented recently
